utils/bluez-tools/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2021 Karl Osterseher <karli_o@gmx.at>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bluez-tools
+PKG_VERSION:=20201025.f653217
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
+
+PKG_MAINTAINER:=Alex Orlenko <none>, Karl Osterseher <karli_o@gmx.at>
```

Comment:
I think we also need a valid email address for the maintainer **Alex Orlenko**
The maintainer is needed or should be indicated who feels responsible for the package in OpenWrt and can therefore be contacted if there are problems with the package. Therefore a valid email address should be entered here, or you remove him.